code/modules/clothing/glasses/_glasses.dm

```diff
@@ -540,6 +540,10 @@
 	vision_flags = SEE_TURFS
 	var/list/hudlist = list(DATA_HUD_MEDICAL_ADVANCED, DATA_HUD_DIAGNOSTIC_ADVANCED, DATA_HUD_SECURITY_ADVANCED)
 	var/xray = FALSE
+	UNACIDABLE
+	LAVA_PROOF
+	FREEZE_PROOF
+	INDESTRUCTIBLE
```

Comment:
Remove these and add `resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF` underneath `	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE` if you want to make it invincible